pkg/stdlib/html/scroll_top.go

```diff
@@ -23,5 +24,17 @@ func ScrollTop(ctx context.Context, args ...core.Value) (core.Value, error) {
 		return values.None, err
 	}
 
-	return values.None, doc.ScrollTop(ctx)
+	var opts drivers.ScrollOptions
+
+	if len(args) > 1 {
+		opts, err = toScrollOptions(args[1])
+
+		if err != nil {
+			return values.None, err
+		}
+	} else {
```

Comment:
`opts` already is `drivers.ScrollOptions{}` before `if len(args) > 1`. Therefore, this branch can be removed.